src/layouts/EditHomepage/EditHomepage.jsx

```diff
@@ -267,6 +292,32 @@ const EditHomepage = ({ match }) => {
             sectionsErrors.push({ infopic: getErrorValues(INFOPIC_SECTION) })
           }
 
+          if (section.announcements) {
+            sectionsErrors.push({
+              announcements: getErrorValues(ANNOUNCEMENT_BLOCK),
+            })
+            announcementItemErrors = _.map(
+              section.announcements.announcement_items,
+              () => getErrorValues(ANNOUNCEMENT_SECTION)
+            )
+            if (!section.announcements.announcement_items) {
+              // define an empty array to announcement_items to prevent error
```

Comment:
Just checking, will we allow for having the announcements block without any announcement item?

Reply:
yes this is the case